django/contrib/admin/options.py

```diff
@@ -1648,23 +1652,16 @@ def response_action(self, request, queryset):
 
             response = func(self, request, queryset)
 
+            self.message_user(request, "Action executed successfully", messages.SUCCESS)
```

Comment:
🎗️ For sure it could be better message or more detailed. I'll wait for comments!